other/upgrade_2-1_mysql.sql

```diff
@@ -1289,7 +1289,7 @@ CREATE TABLE IF NOT EXISTS {$db_prefix}mentions (
 	content_id INT DEFAULT '0',
 	content_type VARCHAR(10) DEFAULT '',
 	id_mentioned INT DEFAULT 0,
-	id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
+	id_member MEDIUMINT(10) UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
size (10) is here wrong -> remove.

Reply:
MEDIUMINT does in fact support display width, see:
https://dev.mysql.com/doc/refman/5.7/en/numeric-type-overview.html

Reply:
nope doesn't supprt,
it's only ingored so that some person can do mistake without harming ther self:
http://sqlfiddle.com/#!9/51eed/1/0

Reply:
when you look across the installer file,
you find no place where mediumint got a size value.
So you leave the existing consistency.

Please keep in mind that the structure of the installer is leading and
upgrader script needs to follow.

Reply:
Yes.... Clearly, SOMEONE went thru and deleted them all...

There are a bunch across the upgrader.  And I can't find any in my old 2.0 environment.  

For consistency, I'll remove all display widths on all mediumints.